go.work.example

```diff
@@ -1,17 +0,0 @@
-go 1.19
-
-use (
-	.
-	./api
-	./client/v2
-	./core
-	./tools/cosmovisor
-	./depinject
-	./errors
-	./math
-	./orm
-	./tx
-	./simapp
-	./tests
-	./tools/rosetta
```

Comment:
Why is it deleted?
Some people want to use go.work so we provide an already made go.work as an example. We should just add ./store here.

If we add a go.mod, we should as well add it to dependabot.